examples/browser-api-playground/src/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useEffect } from "react";
```

Comment:
Why are we importing `useEffect` here?

Reply:
left from a previous version that trying to do the injection at here instead

Reply:
of course, in the next push ;)